rules/escalate-done-videos/rule.md

```diff
@@ -13,32 +13,30 @@ redirects: []
 
 ---
 
-Done videos provide a great way to get quick feedback on your work However, it is important that these videos get escalated to the right people for feedback.
+We use various project management systems to manage our projects. Key updates on these platforms often include Done Videos, critical text additions, or specification documents.
```

Comment:
I get the idea what you mean with "key updates"... but we would normally refer to "priority updates" or "priority tasks" that the PO requested and moved up higher in the sprint (with their justifications of course). And this is usually a good prompt for the Dev team to ask the PO for the "priorities" to be requested in an email that we can reply to.